src/main.js

```diff
@@ -78,6 +79,13 @@ async function run() {
       await createRefOnGitHub(versionToBuild);
     }
 
+    if (createRef && includeMajorRelease && !calculatePrereleaseVersion) {
+      const majorVersion = versionToBuild.split('.')[0];
+      await createRefOnGitHub(majorVersion);
```

Comment:
Here we might want to consider "updating" an existing release instead of creating a new one.  I am not sure yet if this will fail if the major release already exists.  If so, I have an alternative approach.

Reply:
Updating an existing release can create issues with audits, general tracking and ensuring we know what is in production.

Reply:
I haven't looked at the code changes here, but might I suggest that the `createRef` behavior of this action is misplaced anyway? In my opinion, this action should only calculate new `semver`s and shouldn't push any tag nor create any release in GitHub.

So, given that, I think I'd suggest this action could be extended to have an additional output or two such as `NEXT_MAJOR_VERSION` and possibly `NEXT_MAJOR_MINOR_VERSION`. And some other action can either push a tag or create a release as desired.

Reply:
> Updating an existing release can create issues with audits, general tracking and ensuring we know what is in production.

So this is only updating a release of a major version `v1` where it will still create the deafult release `v1.1.0`.  The `v1` release evolves as new default releases are generated.

`v1` = `v1.1.1`
`v1` = `v1.1.2`.

GitHub's action library uses this same pattern as found in their internal action:
https://github.com/actions/publish-action/blob/main/src/api-utils.ts#L93

Does that satisify your concern over issues with audits @dannybarrus ?

Reply:
> I haven't looked at the code changes here, but might I suggest that the `createRef` behavior of this action is misplaced anyway? In my opinion, this action should only calculate new `semver`s and shouldn't push any tag nor create any release in GitHub.
> 
> So, given that, I think I'd suggest this action should only be extended to have an additional output or two such as `NEXT_MAJOR_VERSION` and possibly `NEXT_MAJOR_MINOR_VERSION`, while some other action can either push a tag or create a release as desired.

I will create a new action that handles this then and repurposes this PR to only output the outputs you suggested @mike-schenk .

Reply:
If indeed this action should not create the release, should it export the sha referenced?